osxphotos/cli/__init__.py

```diff
@@ -76,19 +74,24 @@
 from .list import _list_libraries, list_libraries
 from .orphans import orphans
 from .persons import persons
-from .photo_inspect import photo_inspect
 from .places import places
-from .query import query
 from .repl import repl
-from .show_command import show
 from .snap_diff import diff, snap
-from .sync import sync
 from .theme import theme
-from .timewarp import timewarp
 from .tutorial import tutorial
-from .uuid import uuid
 from .version import version
 
+if is_macos:
+    from .add_locations import add_locations
+    from .batch_edit import batch_edit
+    from .import_cli import import_cli
+    from .photo_inspect import photo_inspect
+    from .query import query
```

Comment:
Why is query gated here? query should be able to run on linux just like export with the exception of `--selected`.  The `--selected` option should also be gated in `cli_params.py` so it's not available on linux. (It is used by both export and query)

Reply:
For some reason I thought `query` relied more on `photoscript` than it does. Fixed. Thanks also for the reminder on `--selected`.